pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -196,6 +198,51 @@ public void setupHelixEntries(final String topicName, final String realtimeTable
         nReplicas);
   }
 
+  // Remove all trace of LLC for this table.
+  public void cleanupLLC(final String realtimeTableName) {
+    // Start by removing the kafka partition assigment znode. This will prevent any new segments being created.
+    ZKMetadataProvider.removeKafkaPartitionAssignmentFromPropertyStore(_propertyStore, realtimeTableName);
+    LOGGER.info("Removed Kafka partition assignemnt (if any) record for {}", realtimeTableName);
+    // If there are any completions in the pipeline we let them commit.
+    Uninterruptibles.sleepUninterruptibly(1, TimeUnit.SECONDS);
```

Comment:
Is there a rationale behind the 1 second number? Are there situations where 1 second wouldn't be enough (eg. full gc on server or controller for several seconds)?